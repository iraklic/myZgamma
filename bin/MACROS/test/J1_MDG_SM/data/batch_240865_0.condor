000 (240865.000.000) 12/15 10:50:37 Job submitted from host: <131.225.191.196:59123>
...
001 (240865.000.000) 12/15 10:51:27 Job executing on host: <131.225.206.189:54644>
...
006 (240865.000.000) 12/15 10:52:35 Image size of job updated: 80980
...
005 (240865.000.000) 12/15 12:47:05 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 01:55:22, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:55:22, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	56163  -  Run Bytes Sent By Job
	1420436  -  Run Bytes Received By Job
	56163  -  Total Bytes Sent By Job
	1420436  -  Total Bytes Received By Job
...
