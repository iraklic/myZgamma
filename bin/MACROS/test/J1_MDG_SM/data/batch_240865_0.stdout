
CMSSW on Condor

started at Thu Dec 15 10:51:28 CST 2011

parameter set:
CONDOR_CLUSTER: 240865
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_data.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_240865_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_240865_0.log 2>&1\n

finished at Thu Dec 15 12:47:05 CST 2011
